build.gradle

```diff
@@ -23,3 +24,53 @@ dependencies {
 tasks.named('test') {
 	useJUnitPlatform()
 }
+
+subprojects {
+	group = 'com.jyc'
```

Comment:
what does jyc means?

Reply:
Jeongpil nim, Yuchan nim, and ChaeLyeon ... 😄